src/index.ts

```diff
@@ -0,0 +1,29 @@
+import buildApp from './app';
+import { PORT, KEYCLOAK_URL } from './env';
+
+process.on('uncaughtException', (err) => {
+    console.log(`Uncaught Exception: ${err.message}`);
+    process.exit(1);
+});
+
+process.on('unhandledRejection', (reason, promise) => {
+    console.log('Unhandled rejection at ', promise, `reason: ${reason}`);
+    process.exit(1);
+});
+
+process.on('SIGINT', () => {
+    console.log(`Process ${process.pid} has been interrupted`);
+    process.exit(0);
+});
+
+const keycloakConfig = {
```

Comment:
would it be better to extract it into its own file? But maybe it would be overkilled.